src/modules/getData.js

```diff
@@ -0,0 +1,9 @@
+const getData = async () => {
+  const response = await fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/0C2VkIG8dHneddUlh9IB/scores/',
+  );
+  const endScore = await response.json();
+  return endScore.result;
+};
+
```

Comment:
[REQUIRED] Data is not displayed on the leaderboard when I submit or click refresh.

![leaderb](https://user-images.githubusercontent.com/62938735/217788303-732452c9-dc99-431a-935a-e55c4688a77d.gif)

Reply:
@fobadara When I run code on my side it is working I don't know it is not working on your side

Reply:
Could you please run build because on my side everything is working properly @fobadara